packages/webpack-cli/lib/groups/AdvancedGroup.js

```diff
@@ -13,21 +13,12 @@ class AdvancedGroup extends GroupHelper {
         if (args.hot) {
             const { HotModuleReplacementPlugin } = require('webpack');
             const hotModuleVal = new HotModuleReplacementPlugin();
-            if (options && options.plugins) {
-                options.plugins.unshift(hotModuleVal);
-            } else {
-                options.plugins = [hotModuleVal];
-            }
+            options.plugins = [hotModuleVal];
         }
         if (args.prefetch) {
             const { PrefetchPlugin } = require('webpack');
             const prefetchVal = new PrefetchPlugin(null, args.prefetch);
-            if (options && options.plugins) {
-                options.plugins.unshift(prefetchVal);
-            } else {
-                // Currently the Plugin function is not functional  -> https://github.com/webpack/webpack-cli/pull/1140#discussion_r376761359
-                // options.plugins = [prefetchVal];
-            }
```

Comment:
@rishabh3112  I think we just need to uncomment this, I think he meant to comment `args.plugin` check here [https://github.com/webpack/webpack-cli/pull/1140#discussion_r376761359](https://github.com/webpack/webpack-cli/pull/1140#discussion_r376761359)

Reply:
Yes, he meant `args.plugin` there.